packages/console/src/scss/_dimensions.scss

```diff
@@ -6,3 +6,5 @@ $modal-layout-grid-large: 850px;
 $modal-layout-grid-medium: 668px;
 $modal-layout-grid-small: 500px;
 $form-text-field-width: 556px;
+$tenant-dropdown-max-width: 500px;
+$tenant-dropdown-min-width: 320px;
```

Comment:
IMO these variables are specific to tenant dropdown, right? Then it's OK to me to write it directly in its component scss file, since they are not reusable by the other components